src/main/features/claim/helpers/featuresBuilder.ts

```diff
@@ -9,6 +9,10 @@ const claimStoreClient: ClaimStoreClient = new ClaimStoreClient()
 const featureTogglesClient: FeatureTogglesClient = new FeatureTogglesClient()
 
 export class FeaturesBuilder {
+  static readonly MEDIATION_PILOT_AMOUNT = 500
```

Comment:
Shouldnt we be moving these variables to Config files, so we can easily change limits without code changes?

Reply:
a config file would still be code and require a deployment. the things that wouldn't need a deployment would be a secret and then there is less visibility on current values and who/when they changed.